src/main/java/ch/njol/skript/events/SimpleEvents.java

```diff
@@ -708,7 +710,23 @@ public class SimpleEvents {
 					.since("2.7")
 					.requiredPlugins("Paper 1.16+");
 		}
-
+		Skript.registerEvent("Entity Breed", SimpleEvent.class, EntityBreedEvent.class, "[entity] breed[ing]")
+			.description("Called whenever two breedable entities begin to conceive a child.")
+			.examples(
+				"on breeding:",
+				"\tsend \"When a %breeding mother% and %breeding father% love each other they make %offspring%\" to breeder"
+			)
+			.since("INSERT VERSION");
```

Comment:
```suggestion
				.description("Called whenever two breedable entities begin to conceive a child.")
				.examples(
					"on breeding:",
						"\tsend \"When a %breeding mother% and %breeding father% love each other they make %offspring%\" to breeder"
				)
				.since("INSERT VERSION");
```